clang/tools/clang-format/clang-format-diff.py

```diff
@@ -185,6 +191,8 @@ def main():
             diff_string = "".join(diff)
             if len(diff_string) > 0:
                 sys.stdout.write(diff_string)
+                if args.non_zero_exit_code:
+                    sys.exit(1)
```

Comment:
```suggestion
                sys.exit(1)
```
I don't think we need to add an option for this.

Reply:
But it may be a breaking change for others if it suddenly does not return 0.

Reply:
I was aware of that but doubt it woud break any valid use cases. In fact, we made similar fixes to git-clang-format without adding options. We can always revisit this if we get valid complaints.